openslides/users/static/js/users/users.js

```diff
@@ -30,6 +36,14 @@ angular.module('OpenSlidesApp.users', [])
             views: {
                 '@users.user': {}
             }
+        })
+        .state('users.user.csv-import', {})
```

Comment:
After the merge of #1498 the id has to be an integer. So you can put the csv-import at any place and do not have to define it before the detail view